sirepo/package_data/static/js/raydata.js

```diff
@@ -502,6 +513,22 @@ SIREPO.app.directive('scansTable', function() {
                 hoveredIndex = index;
             };
 
+            $scope.runAnalysis = (scan, event) => {
+                event.stopPropagation();
+                requestSender.sendStatelessCompute(
+                    appState,
+                    (json) => {},
+                    {
+                        method: 'run_analysis',
+                        args: {
+                            catalogName: appState.models.catalog.catalogName,
+                            uid: scan.uid,
```

Comment:
`uid` is another confusing value. It has nothing to do with our normal `uid` values. Too much to change, but reading the code is confusing for a newbie.